dataeng/resources/load-insights-to-vertica.sh

```diff
@@ -3,11 +3,8 @@
 env
 
 ${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
- LoadAllVideoToVertica --local-scheduler \
- --date $(date +%Y-%m-%d -d "$TO_DATE") \
+ LoadInsightsTableToVertica --local-scheduler \
  --schema $SCHEMA \
  --credentials $CREDENTIALS \
  --marker-schema $MARKER_SCHEMA \
- --overwrite \
- $EXTRA_ARGS
```

Comment:
nit: just wondering if we want to leave $EXTRA_ARGS on all jobs as a matter of principle.